projectgen-micronaut-openrewrite/src/main/resources/META-INF/rewrite/microstream-rest.yml

```
@@ -0,0 +1,35 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: io.micronaut.starter.feature.microstream-rest
+displayName: Add Microstream Rest dependency and link to README
+recipeList:
+  - io.micronaut.starter.feature.microstream-rest.dependencies
+  - io.micronaut.starter.feature.microstream-rest.micronaut.documentation
+  - io.micronaut.starter.feature.microstream-rest.documentation.thirdparty
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: io.micronaut.starter.feature.microstream-rest.dependencies
+displayName: Add Microstream Rest dependency
+recipeList:
+  - org.openrewrite.java.dependencies.AddDependency:
+      groupId: io.micronaut.microstream
+      artifactId: micronaut-microstream-rest
+      configuration: devlopmentOnly
```

There is a typo here: 
```suggestion
      configuration: devlopmentOnly
```